packages/diagrams/backend/sirius-components-diagrams/src/main/java/org/eclipse/sirius/components/diagrams/description/NodeDescription.java

```diff
@@ -278,6 +287,8 @@ public static final class Builder {
 
         private Function<VariableManager, IStatus> dropNodeHandler;
 
+        private Map<String, BorderNodePosition> initialChildBorderNodePositions = new LinkedHashMap<>();
```

Comment:
Same comment

Reply:
I reckoned it was easier to initialize an empty map that allows not being forced to call the method on the builder when it does not make sense. (A NodeDescription does not necessarily have a border node or the specifier does not want to define an initial position).

Note that it is not the only field that is initialized `List<String> reusedBorderNodeDescriptionIds = new ArrayList<>();`
so is it really what you want?